modules/distribution/resources/migration-1.8.0_to_1.9.0/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/util/ResourceUtil.java

```diff
@@ -0,0 +1,278 @@
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.wso2.carbon.apimgt.impl.APIConstants;
+import org.wso2.carbon.apimgt.impl.utils.APIMgtDBUtil;
+import org.wso2.carbon.apimgt.migration.APIMigrationException;
+import org.wso2.carbon.apimgt.migration.client.MigrationDBCreator;
+import org.wso2.carbon.registry.core.RegistryConstants;
+import org.wso2.carbon.utils.CarbonUtils;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerConfigurationException;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class ResourceUtil {
+
+    private static final Log log = LogFactory.getLog(ResourceUtil.class);
+
+    /**
+     * location for the swagger 1.2 resources
+     *
+     * @param apiName     api name
+     * @param apiVersion  api version
+     * @param apiProvider api provider
+     * @return swagger v1.2 location
+     */
+    public static String getSwagger12ResourceLocation(String apiName, String apiVersion, String apiProvider) {
+        return APIConstants.API_DOC_LOCATION + RegistryConstants.PATH_SEPARATOR + apiName + "-" + apiVersion + "-"
+                + apiProvider + RegistryConstants.PATH_SEPARATOR + APIConstants.API_DOC_1_2_LOCATION;
+    }
+
+
+    /**
+     * location for the swagger v2.0 resources
+     *
+     * @param apiName     name of the API
+     * @param apiVersion  version of the API
+     * @param apiProvider provider name of the API
+     * @return swagger v2.0 resource location as a string
+     */
+    public static String getSwagger2ResourceLocation(String apiName, String apiVersion, String apiProvider) {
+        return APIConstants.API_ROOT_LOCATION + RegistryConstants.PATH_SEPARATOR + apiProvider
+                + RegistryConstants.PATH_SEPARATOR + apiName + RegistryConstants.PATH_SEPARATOR + apiVersion
+                + RegistryConstants.PATH_SEPARATOR + "swagger.json";
+    }
+
+
+    /**
+     * This method is used to get the database driver name
+     *
+     * @return user database type as a string
+     * @throws SQLException
+     */
+    private static String getDatabaseDriverName() throws SQLException {
+        Connection connection = APIMgtDBUtil.getConnection();
+        String databaseType;
+
+        if (connection.getMetaData().getDriverName().contains("MySQL")) {
+            databaseType = "MYSQL";
```

Comment:
Removed the method getDatabaseDriverName() in commit  f4f7e4de35c06be8e4da1deffc847cefe3701ac6
We use MigrationDBCreator.getDatabaseType for getting the database type